servers/lumen/routes/web.php

```diff
@@ -243,4 +243,23 @@
             ->header('Access-Control-Allow-Credentials', 'true')
             ->header('Connection', 'keep-alive');
     });
+
+     // DIAGNOSIS
+     $router->get('diagnosis/v20/', ['uses' => 'DiagnosisController@getAllTemporalDiagnosis']);
+     $router->get('diagnosis/v20/{serverSideRowUuid}', ['uses' => 'DiagnosisController@getOneDiagnosis']);
+     $router->post('diagnosis/v20/', ['uses' => 'DiagnosisController@create']);
+     $router->delete('diagnosis/v20/{serverSideRowUuid}', ['uses' => 'DiagnosisController@delete']);
+     $router->patch('diagnosis/v20/{serverSideRowUuid}', ['uses' => 'DiagnosisController@delete']);
```

Comment:
Line no: 252. -> Need to remove. Useless method

Reply:
remove 'getOneDiagnosis' useless rout